lib/exercises.js

```diff
@@ -1,33 +1,87 @@
 // This method will return an array of arrays.
 // Each subarray will have strings which are anagrams of each other
-// Time Complexity: ?
-// Space Complexity: ?
+// Time Complexity: o(n)
+// Space Complexity: o(n)
 function grouped_anagrams(strings) {
```

Comment:
👍 Assuming the words are english words (limited in length) the Big-O is correct.